commonslibrary/urls.py

```diff
@@ -14,6 +14,7 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^accounts/', include('accounts.urls')),
+    url(r'pages/', include('pages.urls')),
```

Comment:
Don't think you need this either?

Reply:
Why? I follow the way django.contrib.flatpages.urls does.